service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/workspace/CloneAllResourcesFlight.java

```diff
@@ -108,6 +108,11 @@ private void addFlightLaunchStepsForResource(
                     resourceCloneInputs.getDestinationResourceId(),
                     resourceCloneInputs.getDestinationFolderId()),
                 RetryRules.shortDatabase());
+            addStep(
+                new AwaitCloneControlledFlexibleResourceFlightStep(
+                    resource.castByEnum(WsmResourceType.CONTROLLED_FLEXIBLE_RESOURCE),
+                    resourceCloneInputs.getFlightId()),
+                RetryRules.shortDatabase());
```

Comment:
No cloud operations for flex resources. So, use `RetryRules.shortDatabase()` instead of `RetryRules.cloudLongRunning()`.